### DISCUSSION TITLE
nodejs version for debian 10

### DISCUSSION THREAD

**Discussion by student-CE** [mastodon/mastodon#22634] [Q&A]
Hi, I want to install **Mastodon** **v4.0.2** on a linux server """**debian 10**""", anyone could advice me which version of nodejs i should install for it ?

**nodejs version 8 or version 16 ?????!!!!**

**Comment by mjankowski** [comment]
I realize this is three years later (and thus there are multiple newer releases) -- but, generally speaking, the supported node/ruby/etc versions are included in the release notes for any particular version.
